pkg/metrics/statsd/statsd.go

```diff
@@ -21,23 +22,25 @@ type statsd struct {
 	log *logrus.Entry
 	env env.Core
 
-	account   string
-	namespace string
+	account      string
+	namespace    string
+	mdmsocketEnv string
 
 	conn net.Conn
 	ch   chan *metric
 
 	now func() time.Time
 }
 
-// New returns a new metrics.Emitter
-func New(ctx context.Context, log *logrus.Entry, env env.Core, account, namespace string) metrics.Emitter {
+// New returns a new metrics.Interface
```

Comment:
```suggestion
// New returns a new metrics.Emitter
```
This function still returns a `metrics.Emitter`